src/components/Charts/ChartContactTracing.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ChartZones from './ChartZones';
+import { CONTACT_TRACING_LEVEL_INFO_MAP } from 'common/metrics/contact_tracing';
+import { formatPercent } from './utils';
+
+const CAP_Y = 1;
+
+const getPointText = (valueY: number) => formatPercent(valueY, 0);
+
+const getTooltipText = (valueY: number) =>
```

Comment:
do we need to do the capping stuff we did in icu headroom?

Reply:
I'm assuming that we can't go over 100% contact tracing. For ICU headroom, there are some cases that go over 100% (I checked on the `/all` page), although I don't know how that works in real life 🤷‍♂️